src/testdrive/src/main.rs

```diff
@@ -119,7 +120,14 @@ async fn run(args: Args) -> Result<(), Error> {
                 );
                 (region, account.into(), credentials)
             }
-        };
+        }
+    };
+
+    if args.files.iter().any(|f| f.contains("esoteric")) {
```

Comment:
Hardcoding paths like this violates separation of concerns. testdrive the binary is used in more than just the `test/testdrive/mzcompose.yml` configuration; it should not contain assumptions about the behavior of tests based upon their filenames.

If you think it's important, I'd endorse unconditionally printing out the non-sensitive configuration parameters here instead.